Practise_Problems/orderingSystem.js

```diff
@@ -1,114 +1,115 @@
-//Structure of Commands : ['INSERT', 'Item', 'Price'] or ['VIEW', '-', '-']
-
-const machine = (commands) => {
-  //Structure of the store = [  [[item1, item2, item3], price]  ];
-  let store = [];
-  let visited = 0;
-  let result = [];
-
-  for (let i = 0; i < commands.length; i++) {
-    if (commands[i][0] == "INSERT") {
-      let price = commands[i][2];
-      let item = commands[i][1];
-
-      if (store.length > 0) {
-        for (let j = 0; j < store.length; j++) {
-          //A value to keep track of store indexes
-          let tracker = 0;
-
-          //If new item is cheaper than the index. Unshift
-          if (store[j][1] > price) {
-            let firstHalf = store.slice(0, j);
-            let secondHalf = store.slice(j);
-
-            firstHalf.push([[item], price]);
-            store = firstHalf.concat(secondHalf);
-            break;
-          }
-
-          //If new item is equal to the index. Add to the array, alphabetically
-          else if (store[j][1] === price) {
-            store[j][0].push(item);
-
-            //sort by name
-            //store[j][0].sort();
-            break;
-          }
+  //Structure of Commands : ['INSERT', 'Item', 'Price'] or ['VIEW', '-', '-']
+  // a change
+
+  const machine = (commands) => {
+    //Structure of the store = [  [[item1, item2, item3], price]  ];
+    let store = [];
+    let visited = 0;
+    let result = [];
+
+    for (let i = 0; i < commands.length; i++) {
+      if (commands[i][0] == "INSERT") {
+        let price = commands[i][2];
+        let item = commands[i][1];
+
+        if (store.length > 0) {
+          for (let j = 0; j < store.length; j++) {
+            //A value to keep track of store indexes
+            let tracker = 0;
+
+            //If new item is cheaper than the index. Unshift
+            if (store[j][1] > price) {
+              let firstHalf = store.slice(0, j);
+              let secondHalf = store.slice(j);
+
+              firstHalf.push([[item], price]);
+              store = firstHalf.concat(secondHalf);
+              break;
+            }
 
-          //If new item is more expensive than index.
-          else {
-            while (tracker <= store.length) {
-              //End of the store and no more items to compare with
-              if (tracker === store.length) {
-                store.push([[item], price]);
-                break;
-              }
+            //If new item is equal to the index. Add to the array, alphabetically
+            else if (store[j][1] === price) {
+              store[j][0].push(item);
```

Comment:
question: how does the alphabetically sort work? or is the comment above outdated?